docs/install.md

```diff
@@ -60,7 +60,16 @@ To install Compose, do the following:
 
 ### Install using pip
 
-    $ sudo pip install -U docker-compose
+Compose can be installed from [pypi](https://pypi.python.org/pypi/docker-compose)
+using `pip`.  If you install using `pip` it is highly recommended that you use a
+[virtualenv](https://virtualenv.pypa.io/en/latest/) because many operating systems
+have python system packages that conflict with docker-compose dependencies.
```

Comment:
I think I'd rather we link them to elsewhere to learn about it, that'll take them through the setup steps which already includes explanation so they can learn why they're doing those steps. Like maybe http://docs.python-guide.org/en/latest/dev/virtualenvs/ this is a good one as it also includes the additional tool of `virtualenvwrapper`.